lntest/itest/log_error_whitelist.txt

```diff
@@ -188,3 +188,4 @@
 <time> [ERR] FNDG: Unable to advance state(<chan_point>): failed adding to router graph: error sending channel announcement: gossiper is shutting down
 <time> [ERR] PEER: unable to close channel, ChannelID(<hex>) is unknown
 <time> [ERR] HSWC: ChannelLink(<chan>): unable to update signals
+<time> [ERR] RPCS: [/routerrpc.Router/HtlcInterceptor]: rpc error: code = Canceled desc = context canceled
```

Comment:
If this is newly introduced, it may be worth to log context canceled specifically as warn. A tiny step towards getting rid of all errors that aren't really errors.

Reply:
This is not something added explicitly. I think there is somewhere a code that logs every RPC call that returns with error. In this case the error is the context cancellation when the stream is closed.

Reply:
Ok, out of scope for this pr then